pkg/mixin/remote-mixins/index.json

```diff
@@ -0,0 +1,79 @@
+[
```

Comment:
I know you've been kicking around having less duplicate information and just the entry for the feed or source url here. I'm not sure how much we need to optimize out of the gate. For now this works great, and the only burden is on us to list new mixins.

Longer term, it would be nice if we could just list our atom.xml and not list each mixin for example. Even longer term, I would assume if we are wildly sucessful beyond our dreams, support for localization would come up.

I think this file format is just fine for now though.

Reply:
Yes, as we've discussed, _if_ we decide to mandate that an atom feed URL be available for any porter mixin intended for broadcast (ideally we'd supply/demonstrate a quick/easy way to host said feed), all of the metadata we'd need could potentially be sourced from the feed XML.  If we go this route, we may want to add a few more fields to our atom feed template; for example, a `subtitle` or `summary` field which could hold a description of the plugin, etc.

Reply:
A summary would be really nice 👍